README.md

```diff
@@ -729,6 +729,14 @@ To set a global theme, you can do the following in your app delegate:
 [Chameleon setGlobalThemeUsingPrimaryColor:(UIColor *)color withContentStyle:(UIContentStyle)contentStyle];
 ```
 
+######Swift
+``` swift
+Chameleon.setGlobalThemeUsingPrimaryColor(.flatPlum(),
```

Comment:
Update to 8.1. I've just cloned the repo and it works without the () (as expected)